app/src/main/java/com/intive/selftraining/selftraining/network/models/movieDetails/BelongsToCollectionEntity.kt

```diff
@@ -0,0 +1,8 @@
+package com.intive.selftraining.selftraining.network.models.movieDetails
+
+data class BelongsToCollectionEntity(
```

Comment:
This class is named BelongsToCollectionEntity because the field in the api endpoint is called belongs_to_collection ?

Reply:
yes, you are right